domains/mubarak.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "link to Mubarak H. Alketbi Github profile",
+  "repo": "https://github.com/MubarakHAlketbi",
+  "owner": {
+    "username": "MubarakHAlketbi",
+    "email": "mubarak.harran@gmail.com"
+  },
+  "record": {
+    "URL": "https://mubarakhalketbi.com"
```

website redirects to your github profile
```suggestion
    "URL": "https://github.com/MubarakHAlketbi/"
```